src/participants/create_project.py

```diff
@@ -134,10 +138,10 @@ def handle_wi(self, wid):
             return
 
         # events for official projects that are gated get diverted to a side project
-        prjobj = get_or_none(Project, name=project, obs__apiurl=self.obs.apiurl)
+        prjobj = Project.get_matching(project, self.obs.apiurl)
         if prjobj and prjobj.gated:
-            print(prjobj, "is gated")
-            link = project
+            print "%s is gated" % prjobj
```

Comment:
Aha, I was confused by Pami's recent tendency to `from __future__ import print_funcction` (mostly for line-wrapping arguments).  Here it was actually a print statement actting on a 2-tuple.  Nothing to see here :)